vllm/model_executor/model_loader/weight_utils.py

```diff
@@ -54,6 +54,29 @@
         "SafeTensorsFileLoader")
     SingleGroup = fastsafetensors.placeholder_attr("SingleGroup")
 
+import importlib
+from importlib.util import find_spec
+
+from packaging import version
+
+
+def check_torchao_safetensors_availability():
```

Comment:
I think we can change this to a more general check, e.g. `torchao_version_at_least` and then in callsite just use `if torchao_version_at_least("0.14.0")`